src/components/InngestCommHandler.ts

```diff
@@ -524,7 +587,6 @@ export class InngestCommHandler<H extends Handler, TransformedRes> {
                 )
             ),
             headers: {
-              ...getHeaders(),
```

Comment:
> Why were the headers removed? - @goodoldneon 

There was a lot of duplicated code here where headers were set for almost every response.

This is now handled in a single place higher up in `createHandler()` (see [streamed headers](https://github.com/inngest/inngest-js/pull/163/files#diff-5087958ec6f8b087e8827c51631a00210094a1d599c750e7ccade1b12ffc59c5R518) and [regular headers](https://github.com/inngest/inngest-js/pull/163/files#diff-5087958ec6f8b087e8827c51631a00210094a1d599c750e7ccade1b12ffc59c5R527)), then merged with any _extras_ that are passed here, usually just `Content-Type`.